indico/modules/rb_new/operations/bookings.py

```diff
@@ -200,3 +200,10 @@ def get_room_details_availability(room, start_dt, end_dt):
             'day': iso_day,
         })
     return sorted(availability, key=itemgetter('day'))
+
+
+def get_booking_occurrences(booking):
+    candidates = booking.occurrences
```

Comment:
those aren't candidates. you could just use booking.occurrences directly in the next line